openbsd/OpenBSDProcessList.h

```diff
@@ -43,7 +43,7 @@ typedef struct OpenBSDProcessList_ {
 
    CPUData* cpus;
    int cpuSpeed;
-
+   int* cpuIndex;
```

Comment:
Maybe include the index in the `CPUData` struct (the one unused `int` of the average entry is negligible)